src/main/java/com/textmessenger/service/MessageServiceImpl.java

```diff
@@ -67,7 +74,25 @@ public void createMessageWithUserIdDialogId(Long user, Long dialog, String msg)
     message.setContent(msg);
     message.setDialog(userD);
     message.setUser(userM);
-    messageRepository.save(message);
+    Message save = messageRepository.save(message);
+
+    userD.getUsers().forEach(user1 -> {
+      if (user1.getId() != userM.getId()) {
+        TestingWs testingWs = new TestingWs();
+        testingWs.setSender(userM.getLogin());
+        testingWs.setReceiver(user1.getLogin());
+        testingWs.setType("NEW_MESSAGE");
+        testingWs.setMessageToFront(MessageToFront.convertMessageToFront(save));
+        simpMessagingTemplate.convertAndSendToUser(user1.getLogin(), wsPath, testingWs);
+      }
+    });
+
+//    one.getUsers().forEach(u -> {
+//      if (u.getId()!=firstUser.getId()){
+//
+//
+//      }
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=175&issues=AWXEj19jlgbYnohYqVxL&open=AWXEj19jlgbYnohYqVxL)

[ISSUE_KEY]: AWXEj19jlgbYnohYqVxL